Makefile

```diff
@@ -37,8 +37,8 @@ pyflakes:
 	pyflakes pyeapi/ test/
 
 flake8:
-	flake8 --ignore=E201,E202,E302,E303,E402,E731,W391 --exit-zero pyeapi/
-	flake8 --ignore=E201,E202,E302,E303,E402,E731,W391,N802 --max-line-length=100 test/
+	flake8 --ignore=E128,E201,E202,E302,E303,E402,E731,W391 --exit-zero pyeapi/
+	flake8 --ignore=E128,E201,E202,E302,E303,E402,E731,W391,N802 --max-line-length=100 test/
```

Comment:
Why ignore under indentation?

Also is there a reason for adding all of the spaces between the parentheses and other cases?

Reply:
I'm slowly converting to Arista coding style.
1. flake8 indentation check (E128) is ill - it only allows next line continuation which gravitates to the right - which is a bad indentation style, the much better indentation style is when next line indent gravitates to the left (Arista style).
2. spaces between  the parentheses and arguments are also in Arista coding style.

Reply:
Gotcha. Sounds good to me.